### PR TITLE
Handle `AttributeError` returned when element is None

### PR DESCRIPTION
**Pull request by DavisRayM**
### Changes / Features implemented

- Handle `AttributeError` raised for multiple select choice elements

Related to https://sentry.k8s.onalabs.org/organizations/ona/issues/44054/?project=50&query=is%3Aunresolved

### CODE REVIEW

File: onadata/libs/utils/export_builder.py [hunk 1]
```diff
@@ -1207,7 +1207,7 @@ def _is_numeric(xpath, element_type, data_dictionary):
                 # check if it is a choice part of multiple choice
                 # type is likely empty string, split multi select is binary
                 element = data_dictionary.get_element(xpath)
-                if element.type == SELECT_ONE:
+                if element and element.type == SELECT_ONE:
```
Comment (kelvin-muchiri) [inline onadata/libs/utils/export_builder.py:1210]: Should we add a test to ensure the code `is_all_numeric` is not called if element is none?

Comment (DavisRayM) [reply onadata/libs/utils/export_builder.py:1210]: It should be called; the element could be null when the parent is a select multiple

Comment (DavisRayM) [reply onadata/libs/utils/export_builder.py:1210]: The function checks the parent element in the next few lines

Comment (DavisRayM) [reply onadata/libs/utils/export_builder.py:1210]: What scenario would cause it to be called ? I don't think this scenario in particular requires a test...

Comment (DavisRayM) [reply onadata/libs/utils/export_builder.py:1210]: Much like line `1218`; Element are returned as `None` even when in sections in very rare cases... _When the form has a very large choice list e.t.c_

**Review by kelvin-muchiri — APPROVED** [review]
